app/views/posts/index.html.erb

```diff
@@ -1,15 +1,30 @@
 <div>
-    <%= image_tag(@user.photo, width:'150px', height:'200px') %>
-    <h1><%= @user.name %></h1>
-    <p>Number of posts: <%= @user.posts_counter.to_i %></p>
-    <h1>Posts:</h1>
-    <%= link_to "Create a post", new_user_post_path, class: "button" %>
+    <div class="user-con">
+        <%= image_tag(@user.photo, width:'150px', height:'160px', class:'user-img') %>
+        <div class="user-info">
+        <h1><%= @user.name %></h1>
+        <p>Number of posts: <%= @user.posts_counter.to_i %></p>
+        </div>
+    </div>
+    <div class="btn-con">
+        <%= link_to "Create a post", new_user_post_path, class: "post-btn" %>
+    </div>
     <section>
```

Comment:
- [ ] On `posts#create` you should display a flash message if the action is successful or not posts#create that is applied as well on `comments#create`, and `likes#create` actions uses flash messages for a successful and not successful path